.github/workflows/pathogen-repo-ci.yaml

```diff
@@ -237,14 +246,17 @@ jobs:
           fi
 
       - run: nextstrain build . ${{ inputs.build-args }}
+        working-directory: ${{ inputs.workflow-root }}
 
       - if: always()
         uses: actions/upload-artifact@v3
+        env:
+          workdir: ${{ inputs.workflow-root }}
         with:
           name: ${{ inputs.artifact-name }}-${{ matrix.runtime }}
           path: |
-            auspice/
-            results/
-            benchmarks/
-            logs/
-            .snakemake/log/
+            $workdir/auspice/
+            $workdir/results/
+            $workdir/benchmarks/
+            $workdir/logs/
+            $workdir/.snakemake/log/
```

Comment:
This doesn't work. The interpolation does not work like that. If you look at the "successful" test, [you'll see it didn't find any files](https://github.com/nextstrain/monkeypox/actions/runs/6303654582/job/17113366921#step:9:17) and [uploaded no artifacts](https://github.com/nextstrain/monkeypox/actions/runs/6303654582).